Sources/MongoSwift/BSON/Document+Sequence.swift

```diff
@@ -206,24 +206,24 @@ public class DocumentIterator: IteratorProtocol {
     }
 
     /// Returns the current value. Assumes the iterator is in a valid position.
-    internal var currentValue: BsonValue? {
+    internal var currentValue: BSONValue? {
         do {
             switch self.currentType {
             case .symbol:
                 return try Symbol.asString(from: self)
             case .dbPointer:
                 return try DBPointer.asDocument(from: self)
             default:
-                return try DocumentIterator.BsonTypeMap[currentType]?.init(from: self)
+                return try DocumentIterator.BSONTypeMap[currentType]?.init(from: self)
```

Comment:
to be consistent w/ my suggestion below
```suggestion
                return try DocumentIterator.bsonTypeMap[currentType]?.init(from: self)
```